lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -192,10 +192,24 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
   );
 
   const [autoGradingConfig, setAutoGradingConfig] = useState<AutoGradingConfig>(
-    {
-      gradingType: 'all_or_nothing',
-      activityCalculation: 'cumulative',
-      requiredAnnotations: 1,
+    () => {
+      const config = assignment?.auto_grading_config;
```

Comment:
From https://hypothes-is.slack.com/archives/C1M8NH76X/p1728560375109319?thread_ts=1728557569.201379&cid=C1M8NH76X